docs/configuration.md

```diff
@@ -84,17 +84,17 @@ spec:
       kubeSchedulerUser: kube-scheduler
   images:
     konnectivity:
-      image: k8s.gcr.io/kas-network-proxy/proxy-agent
-      version: v0.0.25
+      image: quay.io/k0sproject/apiserver-network-proxy-agent
+      version: 0.0.32-k0s1
     metricsserver:
-      image: k8s.gcr.io/metrics-server/metrics-server
-      version: v0.5.0
+      image: registry.k8s.io/metrics-server/metrics-server
+      version: v0.6.1
     kubeproxy:
-      image: k8s.gcr.io/kube-proxy
+      image: registry.k8s.io/kube-proxy
       version: v1.25.4
     coredns:
-      image: k8s.gcr.io/coredns/coredns
-      version: v1.7.0
+      image: docker.io/coredns/coredns
+      version: 1.9.4
```

Comment:
Do we want to bump up coreDNS this much in a backport?

Reply:
This is the docs, which have been updated in the original PR to match the versions that are actually used by k0s. The 1.25 release is currently on 1.9.4.

Reply:
ah 🤦  all good then :)